libpod/runtime.go

```diff
@@ -362,6 +362,35 @@ func SetXdgRuntimeDir() error {
 	if err := os.Setenv("XDG_RUNTIME_DIR", runtimeDir); err != nil {
 		return errors.Wrapf(err, "cannot set XDG_RUNTIME_DIR")
 	}
+
+	// Setup XDG_CONFIG_HOME
+	cfgHomeDir := os.Getenv("XDG_RUNTIME_DIR")
+
+	cfgHomeDirLinger, err := rootless.EnableLinger()
```

Comment:
we don't need to do this.  We are already enabling linger earlier in the function and the return value is the rundir, not the config dir.

I think we would need only to `util.GetRootlessConfigHomeDir()` and assign its value to the `XDG_CONFIG_HOME`